### ISSUE TITLE
The region is hard-coded in the terraform code.

### ISSUE THREAD

**Issue by coltoneshaw** [mattermost/mattermost-load-test-ng#554]
Add the ability to define what AWS region everything is deployed to. 

https://github.com/mattermost/mattermost-load-test-ng/blob/14168ff28c541c76a6afa03856c904a4aefe14ed/deployment/terraform/assets/cluster.tf#L12-L13

**Comment by coltoneshaw** [comment]
Ideally, this is a config value in the `deployer.json` that the user can control

**Comment by agnivade** [comment]
Sounds reasonable. Would be great if you want to send a PR!

**Comment by coltoneshaw** [comment]
Okay, sticking it here so I can track it. If we want configurable regions, we also need to be able to configure the `ami` since apparently the one used is not in all regions.

```
info  [2023-02-10 12:48:57.533 -05:00] ╷                              caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │ Error: error collecting instance settings: InvalidAMIID.NotFound: The image id '[ami-0fa37863afb290840]' does not exist caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │        status code: 400, request id: da987bbf-ca49-4663-ae5c-0d65f92fa5e3 caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │                              caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │   with aws_instance.app_server[0], caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │   on cluster.tf line 29, in resource "aws_instance" "app_server": caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │   29: resource "aws_instance" "app_server" { caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] │                              caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.533 -05:00] ╵                              caller="terraform/engine.go:75"
info  [2023-02-10 12:48:57.537 -05:00] ╷                              caller="terraform/engine.go:75"
```

**Comment by agarciamontoro** [comment]
Fixed in #560
